engine/src/org/pentaho/di/repository/RepositoryImporter.java

```diff
@@ -1035,4 +1026,32 @@ public void setOverwrite( boolean overwrite ) {
   public String getVersionComment() {
     return versionComment;
   }
+
+  private void saveTransformationToRepo( TransMeta transMeta, RepositoryImportFeedbackInterface feedback )
```

Comment:
@brosander, it seems there is no possibility to do it here because of following reasons:
1) "repo" field 
2)  JobMeta and TransMeta both have .lookupRepositoryReferences and AbstractMeta has not
Maybe I missed something. Thanks